picard/ui/options/scripting.py

```diff
@@ -103,18 +120,142 @@ class ScriptingOptionsPage(OptionsPage):
     ACTIVE = True
 
     options = [
-        config.BoolOption("setting", "enable_tagger_script", False),
-        config.TextOption("setting", "tagger_script", ""),
+        config.BoolOption("setting", "enable_tagger_scripts", False),
+        config.ListOption("setting", "list_of_scripts", []),
```

Comment:
What about upgrades ? A hook should be provided to convert `tagger_script` to first item of `list_of_scripts`.

It can be done in  [picard/config_upgrade.py](https://github.com/metabrainz/picard/blob/master/picard/config_upgrade.py)